go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/google/go-cmp v0.5.6 // indirect
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/regen-network/cosmos-proto v0.3.1
```

Comment:
better if we can import this in `internal/tools.go` with an underscore otherwise code generation will get broken once user deletes the generated .pb file importing this.

Reply:
I didn't understand, which user are you talking about?

Reply:
I mean us. All the proto related dependencies should be added to tools.go if they're not sdk modules.